RELEASE.md

```diff
@@ -99,3 +99,46 @@ Perform the following actions ONLY after the vote is successful and closed.
    svn add {version}
    svn commit -m “Apache Ignite {version}”
    ```
+
+
+## Gradle commands
+1. Fill sign information in gradle.properties file.
+   ```
+   signing.keyId=*INSERT KEY HERE LAST 8 CHARS*
+   signing.password=*INSERT PASSWORD HERE*
+   signing.secretKeyRingFile=*INSERT KEY RING ABSOLUTE PATH HERE* 
+   ```
+   For generate secret key ring file please use follow command
+   ```
+   gpg --keyring secring.gpg --export-secret-keys > ~/.gnupg/secring.gpg
+   ```
+   Show key id command (you need only last 8 chars from printed key)
+   ```
+   gpg -K
+   ```
+2. You can sign any artifact via follow tasks
+   ```
+   signCliZip     --- sign CLI zip distribution 
+   signDbZip      --- sign Ignite zip distribution 
+   signAllDistZip --- sign meta zip distribution (CLI + Ignite)
+   signMavenPublication --- sign jars of all modules
```

Comment:
Could you please clarify which command(s) must be executed by a release manager to build&sing all required artifacts?